### PR TITLE
Fixed GeorepStatusList response

### PR DESCRIPTION
**Pull request by sidharthanup**
GeorepStatusList method should give a list of GeorepSession as the response body but was only giving a session.

Signed-off-by: Sidharth Anupkrishnan <sanupkri@redhat.com>

### CODE REVIEW

File: plugins/georeplication/init.go [hunk 1]
```diff
@@ -101,7 +101,7 @@ func (p *Plugin) RestRoutes() route.Routes {
 			Method:       "GET",
 			Pattern:      "/geo-replication",
 			Version:      1,
-			ResponseType: utils.GetTypeString((*georepapi.GeorepSession)(nil)),
+			ResponseType: utils.GetTypeString((*georepapi.GeorepSessionList)(nil)),
```
Comment (Madhu-1) [inline plugins/georeplication/init.go]: can you update `func getSessionList() ([]*georepapi.GeorepSession, error) {
` function to use `GeorepSessionList`

**Comment by Madhu-1** [comment]
@sidharthanup CI is failing 
```
11:23:26 plugins/georeplication/store-utils.go:87:2: cannot use sessions (type []*"github.com/gluster/glusterd2/plugins/georeplication/api".GeorepSession) as type []*"github.com/gluster/glusterd2/plugins/georeplication/api".GeorepSessionList in return argument
```

**Comment by sidharthanup** [comment]
@Madhu-1 Please check now

**Comment by Madhu-1** [comment]
@sidharthanup still CI is failing
```
11:36:09 plugins/georeplication/store-utils.go:87:2: cannot use sessions (type []*"github.com/gluster/glusterd2/plugins/georeplication/api".GeorepSession) as type *"github.com/gluster/glusterd2/plugins/georeplication/api".GeorepSessionList in return argument
```

please run 
* `make` 
* `make test` 
*  `make functest` 
locally  to make sure everything is okay

**Comment by sidharthanup** [comment]
Changes Done.

**Pull request merged**